### PR TITLE
PR for converting HTML to JSX

### PR DESCRIPTION
**Pull request by AnjaliDeshwani**
**Converted HTML to JSX and Added Router**

### Files to be reviewed -
- Auth
- Cart
- Home
- ProductList
- WishList
- ProductCard

CSS has already been reviewed hence can be skipped.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,20 +24,18 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link
```
Comment (xsaurabhsuthar) [inline public/index.html:27]: replace with <Link> from react-router-dom

File: src/components/Filters/Filters.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        <div className="filter-text">
+          <div className="font-bold">Filters</div>
+          <div>
+            <span className="clear-all-btn font-semibold">Clear all</span>
+          </div>
+        </div>
+        <div className="filter-price">
+          <div className="font-bold">Price</div>
+          <ul className="list stacked-list">
+            <li>
```
Comment (xsaurabhsuthar) [inline src/components/Filters/Filters.jsx:15]: you can extract the repeating component to new component

File: src/components/Footer/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+              target="_blank"
+              rel="noreferrer"
+              className="footer-link"
+            >
+              <i className="fa-brands fa-github"></i>
+            </a>
+          </div>
+
+          <div className="footer-icons">
+            <a
```
Comment (xsaurabhsuthar) [inline src/components/Footer/Footer.jsx:22]: change with <Link>

File: src/components/index.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+export { default as Filters } from "./Filters/Filters";
```
Comment (xsaurabhsuthar) [inline src/components/index.js]: you should do named import and export.

File: src/pages/Auth/SignUp.jsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+              </Link>
+            </div>
+          </div>
+        </div>
+      </main>
+    </>
+  );
+};
+
+export default SignUp;
```
Comment (xsaurabhsuthar) [inline src/pages/Auth/SignUp.jsx]: avoid doing default export.

**Review by xsaurabhsuthar — APPROVED** [review]

**Review by Ash-D23 — APPROVED** [review]
Everything looks good to merge,  also you can start using dummy data and pass it as props for product details

File: src/components/index.js (hunk 4 above)
Comment (AnjaliDeshwani) [reply src/components/index.js]: Thanks Saurabh for reviewing, I have updated it now with name import and export.

**Pull request merged**